script.js

```diff
@@ -217,6 +217,29 @@ function closePopup() {
   }
 }
 
+// local storage code
+function getStorage() {
+  // convert the json string into js object
+  const objFinal = JSON.parse(localStorage.getItem('inputUsuario'));
+
+  document.getElementById('inputUser').value = objFinal.valorUsuario;
+  document.getElementById('inputEmail').value = objFinal.valorCorreo;
+  document.getElementById('textArea').value = objFinal.valorMessage;
+}
+function populateStorage() {
+  const valueUser = document.getElementById('inputUser').value;
+  const valueEmail = document.getElementById('inputEmail').value;
+  const valueMessage = document.getElementById('textArea').value;
+
+  const myObj = {
+    valorUsuario: valueUser,
+    valorCorreo: valueEmail,
+    valorMessage: valueMessage,
+  };
+  // convert object into JSON string and save it in one localStorage variable
+  localStorage.setItem('inputUsuario', JSON.stringify(myObj));
+}
+
```

Comment:
* I suggest you implement the following project required behaviours
- When the user changes the content of any input field, the data is saved to the local storage
- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.
Hint 🤔 You can use a function to check when the page loads if data was found in local storage if true you set it to input value. 👍